retroshare-gui/src/gui/gxsforums/GxsForumThreadWidget.cpp

```diff
@@ -1204,7 +1204,7 @@ void GxsForumThreadWidget::insertMessage()
 
         int current_index = 0 ;
 
-        for(int i=0;i<post_versions.size();++i)
+        for(int i=0;i<static_cast<int>(post_versions.size());++i)

```

Comment:
Please use for(uint32_t i=0;) or for(size_t i=0;) instead of this static_cast. I know it's the same, but it makes the code easier to read.

Reply:
i is used after as integer.
But I can change with older (int) cast.